packages/web/src/components/CheckboxField/_theme.scss

```diff
@@ -0,0 +1,24 @@
+@use '@tokens' as tokens;
+
+$spacing: tokens.$space-300;
+
+$label-typography: tokens.$body-medium-text-regular;
+$label-color-default: tokens.$text-secondary-default;
+$label-color-disabled: tokens.$text-secondary-disabled;
```

Comment:
Please check label colors, it should be `$text-primary-default` and `$text-primary-disabled`.

Reply:
Fixed.